### DISCUSSION TITLE
How do I "brew install dovecot" with MySQL

### DISCUSSION THREAD

**Discussion by RobertChalmers** [Homebrew/discussions#2104] [Everyday usage]
I'm trying to install Dovecot with brew, and it installs fine, but it defaults to sql-lite and I need MySQL
Is there a command line switch I don't know about?
something like "brew install dovecot-mysql" maybe?

BigSur on iMac
Thanks

**Comment by carlocab** [comment]
We only configure Dovecot with sqlite, unfortunately. If you need a version of Dovecot that works with MySQL you can maintain a version of the `dovecot` formula in your own tap that uses MySQL instead.

Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Comment by RobertChalmers** [comment]
Thanks Carlo. Interesting idea. I’ll try it.

Cheers
Robert


-----
Robert Chalmers
https://robert-chalmers.uk
https://robert-chalmers.com
@R_A_Chalmers
